tlklan/protected/extensions/localuser/LocalUser.php

```diff
@@ -61,7 +53,7 @@ public function init()
 	 */
 	public function connect()
 	{
-		$this->_ssh = new Net_SSH2($this->hostname, $this->port);
+		$this->_ssh = new \phpseclib\Net\SSH2($this->hostname, $this->port);
```

Comment:
IMO add a `use` statement instead

Reply:
this is the only place that we use this lib, why add extra lines?

Reply:
It's easier to see from the top off the file what the file uses